dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceServicesIT.java

```diff
@@ -0,0 +1,163 @@
+    private static final String DROPWIZARD_CONFIGURATION_FILE_PATH = CommonTestUtilities.PUBLIC_CONFIG_PATH;
+    public static final DropwizardTestSupport<DockstoreWebserviceConfiguration> SUPPORT = new DropwizardTestSupport<>(
+            DockstoreWebserviceApplication.class, DROPWIZARD_CONFIGURATION_FILE_PATH, ConfigOverride.config("gitHubAppId", GITHUB_APP_ID),
+        ConfigOverride.config("gitHubAppPrivateKeyFile", "./src/test/resources/integrationtest.pem"));
+    public static final long GITHHUB_USER1_ID = 1L;
+    public static final long GITHHUB_USER2_ID = 2L;
+    private static TestingPostgres testingPostgres;
+    @Rule
+    public final ExpectedSystemExit systemExit = ExpectedSystemExit.none();
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    // These are not from Hoverfly, it's actually in the starting database
+    public final static String GITHUB_ACCOUNT_USERNAME_1 = "tuber";
+    public final static String GITHUB_ACCOUNT_USERNAME_2 = "potato";
+    private TokenDAO tokenDAO;
+
+    @Rule
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog().muteForSuccessfulTests();
+    @Rule
+    public final SystemErrRule systemErrRule = new SystemErrRule().enableLog().muteForSuccessfulTests();
+
+    @ClassRule
+    public static final HoverflyRule hoverflyRule = HoverflyRule.inSimulationMode(SERVICES_SIMULATION_SOURCE);
+
+
+    @BeforeClass
+    public static void dropAndRecreateDB() throws Exception {
+        CommonTestUtilities.dropAndRecreateNoTestData(SUPPORT, DROPWIZARD_CONFIGURATION_FILE_PATH);
+        SUPPORT.before();
+        testingPostgres = new TestingPostgres(SUPPORT);
+    }
+
+    @AfterClass
+    public static void afterClass(){
+        SUPPORT.after();
+    }
+
+    @Before
+    public void setup() throws Exception {
+        CommonTestUtilities.dropAndCreateWithTestData(SUPPORT, false, DROPWIZARD_CONFIGURATION_FILE_PATH);
+        DockstoreWebserviceApplication application = SUPPORT.getApplication();
+        SessionFactory sessionFactory = application.getHibernate().getSessionFactory();
+        this.tokenDAO = new TokenDAO(sessionFactory);
+        // non-confidential test database sequences seem messed up and need to be iterated past, but other tests may depend on ids
+        testingPostgres.runUpdateStatement("alter sequence enduser_id_seq increment by 50 restart with 100");
+        testingPostgres.runUpdateStatement("alter sequence token_id_seq increment by 50 restart with 100");
+        // used to allow us to use tokenDAO outside of the web service
+        Session session = application.getHibernate().getSessionFactory().openSession();
+        ManagedSessionContext.bind(session);
+        final Transaction transaction = session.beginTransaction();
+        tokenDAO.create(createToken(SUFFIX1, GITHUB_ACCOUNT_USERNAME_1, GITHHUB_USER1_ID));
+        tokenDAO.create(createToken(SUFFIX2, GITHUB_ACCOUNT_USERNAME_2, GITHHUB_USER2_ID));
```

Comment:
It doesn't look like tokenDAO is used outside of setup. 
Use local scope and even better, should close the transaction and session properly

Reply:
Fixed in upcoming push